<launch>
  <arg name="path"    default="/tmp" />
  <arg name="pattern" default="*" />
  <arg name="ext"     default=".ply" />
  <arg name="rate"    default="1" />
  <arg name="color"   default="false" />
  <arg name="color_max_dist" default="0" />
  <arg name="aligned_frame" default="world_camera" />

  <node name="pc_static_node" pkg="numpy_2_pointcloud" type="PointCloudPub_Static.py" output="screen" required="true">
    <param name="input_dir" value="$(arg path)" />
    <param name="pattern"   value="$(arg pattern)" />
    <param name="ext"       value="$(arg ext)" />
    <param name="rate"      value="$(arg rate)" />
    <param name="color"     value="$(arg color)" />
    <param name="color_max_dist" value="$(arg color_max_dist)" />
    <param name="aligned_frame"  value="$(arg aligned_frame)" />
  </node>
</launch>